tests/ips-state-actions-test-a-01/test.rules

```diff
@@ -0,0 +1,5 @@
+pass udp any any -> any 6081 (sid:1;)
+pass tcp 2a03:b0c0:0002:00d0:0000:0000:0bd3:4001 any -> 2606:2800:0220:0001:0248:1893:25c8:1946 443 (msg:"PASS_CUSTOM_RULE TCP port:443 to support traffic"; flow:established; sid:201000044;)
```

Comment:
This does not apply the `pass` to the flow, but should it? The flow starts as `not_established`, but after transitioning to `established` it stays that way, even if TCP state moves to "closed".

Reply:
That would indeed be coherent with the rule language as we read it.